File: tests/fixtures/sample.java
package com.example;

import java.util.List;
import java.util.ArrayList;

public class SampleService {
    private final List<String> items;

    public SampleService() {
        this.items = new ArrayList<>();
    }

    public void addItem(String item) {
        if (item != null) {
            items.add(item);
        }
    }

    public List<String> getItems() {
        return items;
    }

    public int getItemCount() {
        return items.size();
    }
}

File: tests/fixtures/sample.kt
package com.example

import kotlin.collections.mutableListOf

class SampleRepository {
    private val items = mutableListOf<String>()

    fun add(item: String) {
        items.add(item)
    }

    fun findAll(): List<String> = items.toList()

    fun count(): Int = items.size
}

fun main() {
    val repo = SampleRepository()
    repo.add("hello")
    println(repo.findAll())
}
